marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/steps/CustomStepControllerTest.java

```diff
@@ -29,41 +29,137 @@
     		+ " \"sourceQuery\": \"cts.collectionQuery('default')\" , "
     		+ " \"targetEntityType\": \"http://example.org/Customer-0.0.1/Customer\" , "
     		+ " \"stepDefinitionName\": \"custom\" , "
+            + " \"stepDefinitionType\": \"custom\" , "
+            + " \"stepId\": \"myCustomStep-custom\" , "
     		+ " \"batchSize\": \"50\" , "
     		+ " \"provenanceGranularityLevel\": \"coarse\" "
     		+ "}";
     private final static ObjectMapper MAPPER = new ObjectMapper();
-
-    public static class CustomStep {
-        public String name;
-        public String description;
-        public String selectedSource;
-        public String sourceQuery;
-        public String targetEntityType;
-        public String stepDefinitionName;
-    }
-
-    public static CustomStep newDefaultCustomStep(String name) {
-        CustomStep step = new CustomStep();
-        step.name = name;
-        step.description = "optional";
-        step.selectedSource = "collection";
-        step.sourceQuery = "cts.collectionQuery('test')";
-        step.targetEntityType = "http://example.org/Customer-0.0.1/Customer";
-        step.stepDefinitionName = "custom";
-        return step;
+    
+    @Test
+    void userCanWrite() throws Exception {

```

Comment:
Rename the method names to be consistent with other controller tests (permittedWriteUser())